pkgs/applications/window-managers/jwm/default.nix

```diff
@@ -24,6 +35,8 @@ stdenv.mkDerivation rec {
     '';
 
   meta = {
+    homepage = "http://joewing.net/projects/jwm/";
     description = "A window manager for X11 that requires only Xlib";
+    license = stdenv.lib.licenses.gpl2;
```

Comment:
The meta attributes lacks a `platforms` and `maintainer` field. Would you like to maintain it? If you aren't in the maintainers list yet you can add yourself in `lib/maintainers.nix`.